source/wp-content/themes/wporg-parent-2021/sass/blocks/_spinner.scss

```diff
@@ -0,0 +1,49 @@
+.wporg-spinner {
+	--local-size: 16px;
```

Comment:
Does this need to be a custom property?

Reply:
In the [original](https://github.com/WordPress/wporg-mu-plugins/blob/37e457b27558dd11bc7875a8ab73629572e63e8f/mu-plugins/blocks/google-map/postcss/style.pcss#L78) I was using it for some other calculations and it's been copied over